### ISSUE TITLE
Update Docs

### ISSUE THREAD

**Issue by dlf57** [chemreps/chemreps#59]
- [x] Update Docs

**Comment by dlf57** [comment]
@hienerd and I can work on updating the docs

**Comment by dlf57** [comment]
We currently have an issue with readthedocs where the other pages are not being linked

**Comment by shivupa** [comment]
Yeah I saw this I have a fix I haven't pushed but Ill make a PR tonight

**Comment by KidA3995** [comment]
Hey, I think the contributing page needs to be updated since the precommit hooks have been removed. I'm just pointing this out here rather then submitting another issue because it is another piece of documentation.

**Comment by shivupa** [comment]
Thanks this reminded me about the docs PR.. things got a bit crazy since this post. I'll address them both at once.

**Comment by shivupa** [comment]
Done
